.releaserc.yml

```diff
@@ -0,0 +1,12 @@
+branches:
+  - main
+repositoryUrl: 'git@github.com:liatrio/envelope-game.git'
+preset: conventionalcommits
+plugins:
+  - '@semantic-release/commit-analyzer'
+  - '@semantic-release/release-notes-generator'
+  - path: '@semantic-release/github'
+    failComment: false
+    successComment: false
+
+   
```

Comment:
what this isn't doing:

- bumping `package.json` version and committing back to `main`
- publishing helm chart / bumping `Chart.yaml` version

if you weren't trying to do that, then this is fine.  but I wanted to make sure you knew that this might not be doing everything you want it to.

Reply:
You can use the config file within https://github.com/liatrio/sparky as an example